.readthedocs.yaml

```diff
@@ -0,0 +1,26 @@
+# Read the Docs (RTD) configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD config file version
+version: 2
+
+# Environment RTD sets up for the docs build
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# If using Sphinx, optionally build your docs in additional formats such as PDF
+# formats:
+#    - pdf
+
+# Python requirements required to build the docs
+python:
```

Comment:
i'm not an expert here, just a doubt ....can we do something like this

```

python:
   install:
     - requirements:
         - requirements-base.txt
         - requirements.txt
         - requirements-develop.txt
```

will it work, I see we repeated the requirements, so checking if we could avoid that

Reply:
I double checked the RTD documentation which shows the repetition: https://docs.readthedocs.io/en/stable/config-file/v2.html#requirements-file